client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -27,55 +29,96 @@ import './style.scss';
 
-const connectedProps = ( translate ) => ( {
+const connectedProps = ( translate, connectedMessage ) => ( {
 	iconPath: connectedIcon,
-	className: 'settings_connected',
+	className: 'settings__connected',
 	title: translate( 'Server status: Connected' ),
-	content: translate( 'One-click restores are enabled.' ),
+	content: connectedMessage,
 } );
 
-const disconnectedProps = ( translate ) => ( {
+const disconnectedProps = ( translate, disconnectedMessage ) => ( {
 	iconPath: disconnectedIcon,
-	className: 'settings_disconnected',
+	className: 'settings__disconnected',
 	title: translate( 'Server status: Not connected' ),
-	content: translate(
-		'Enter your server credentials to enable one-click restores for Backups. {{a}}Need help? Find your server credentials{{/a}}',
-		{
-			components: {
-				a: (
-					<ExternalLink
-						className="settings__link-external"
-						icon
-						href="https://jetpack.com/support/adding-credentials-to-jetpack/"
-					/>
-				),
-			},
-		}
-	),
+	content: translate( '%s {{a}}Need help? Find your server credentials{{/a}}', {
+		args: [ disconnectedMessage ],
+		components: {
+			a: (
+				<ExternalLink
+					className="settings__link-external"
+					icon
+					href="https://jetpack.com/support/adding-credentials-to-jetpack/"
+				/>
+			),
+		},
+	} ),
 } );
 
-const ConnectionStatus = ( { isConnected } ) => {
-	const translate = useTranslate();
-	const cardProps = isConnected ? connectedProps( translate ) : disconnectedProps( translate );
-
-	return (
-		<Card compact={ true } className={ cardProps.className }>
-			<img className="settings__icon" src={ cardProps.iconPath } alt="" />
-			<div className="settings__details">
-				<div className="settings__details-head"> { cardProps.title } </div>
-				<div>{ cardProps.content }</div>
-			</div>
-		</Card>
-	);
+const getCardProps = ( isConnected, message, translate ) => {
+	if ( isConnected ) {
+		return connectedProps( translate, message );
+	}
+	return disconnectedProps( translate, message );
 };
 
+const ConnectionStatus = ( { cardProps } ) => (
+	<Card compact={ true } className={ cardProps.className }>
+		<img className="settings__icon" src={ cardProps.iconPath } alt="" />
+		<div className="settings__details">
+			<div className="settings__details-head"> { cardProps.title } </div>
+			<div>{ cardProps.content }</div>
+		</div>
+	</Card>
+);
+
 const SettingsPage = () => {
 	const translate = useTranslate();
 	const siteId = useSelector( getSelectedSiteId );
 
-	const rewind = useSelector( ( state ) => getRewindState( state, siteId ) );
-	const isInitialized = rewind.state !== 'uninitialized';
-	const isConnected = rewind.state === 'active';
+	const HAS_BACKUP = 1;
+	const HAS_SCAN = 2;
+	const HAVE_BOTH = 3;
+
+	const disconnectedMessages = {
+		[ HAS_BACKUP ]: translate(
+			'Enter your server credentials to enable one-click restores for backups.'
+		),
+		[ HAS_SCAN ]: translate(
+			'Enter your server credentials to enable Jetpack to auto fix threats.'
```

Comment:
Do we need to update `auto fix` to `auto-fix`
```suggestion
			'Enter your server credentials to enable Jetpack to auto-fix threats.'
```